cardano-cli/src/Cardano/CLI/Shelley/Key.hs

```diff
@@ -28,15 +43,85 @@ module Cardano.CLI.Shelley.Key
 import           Cardano.Api
 
 import           Control.Monad.IO.Class (MonadIO (..))
+import           Control.Monad.Trans.Except (runExceptT)
+import           Control.Monad.Trans.Except.Extra (handleIOExceptT)
 import           Data.Bifunctor (Bifunctor (..))
+import           Data.ByteString (ByteString)
 import qualified Data.ByteString as BS
+import qualified Data.ByteString.Char8 as BSC
 import qualified Data.List.NonEmpty as NE
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
 
 import           Cardano.CLI.Types
 
+------------------------------------------------------------------------------
+-- Formatted/encoded input deserialisation
+------------------------------------------------------------------------------
+
```

Comment:
Is there a reason this section is empty?

Reply:
Hm, maybe I was planning to re-write some stuff for deserializing encoded keys here. We can probably just remove the section.